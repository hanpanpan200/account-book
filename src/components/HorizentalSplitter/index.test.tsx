import React from 'react';
import { render } from '@testing-library/react';
import HorizontalSplitter from './index';

describe('HorizontalSplitter', () => {
  it('should render correctly', () => {
    const { container } = render(<HorizontalSplitter />);
    expect(container).toMatchSnapshot();
  });
});
