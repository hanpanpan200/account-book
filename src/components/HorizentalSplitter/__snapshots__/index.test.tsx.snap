// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HorizontalSplitter should render correctly 1`] = `
<div>
  <div
    class="container"
  />
</div>
`;
